### PR TITLE
Automatic code cleanup.

### PR DESCRIPTION
**Pull request by ivannaranjo**
This PR includes the result of running the following tools on the code:
* The PowerTools "Organize and Sort Usings" to make sure that we only have usings in our source code for what we actually use.
* The `./tools/ensure_license.py` script, which will make sure that all .cs files have the license header.
* The `./tools/format_files.cmd` script, which will use the [codeformatter]( https://github.com/dotnet/codeformatter) tool to format the source code with pre-defined set of formatting rules.

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/Gae/VersionViewModel.cs [hunk 1]
```diff
@@ -147,7 +147,7 @@ private void OnBrowseStackdriverLogCommand()
                 IsLoading = true;
                 Caption = String.Format(Resources.CloudExplorerGaeMigratingAllTrafficCaption, _version.Id);
 
-                var split = new TrafficSplit { Allocations = new Dictionary<string, double?> { [_version.Id] = 1.0 } };
+                var split = new TrafficSplit { Allocations = new Dictionary<string, double?> {[_version.Id] = 1.0 } };
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/Gae/VersionViewModel.cs]: This spacing looks inconsistent.

File: GoogleCloudExtension/GoogleCloudExtension.DataSources/ResourceManagerDataSource.cs [hunk 2]
```diff
@@ -56,7 +56,8 @@ public ResourceManagerDataSource(GoogleCredential credential, string appName)
         internal ResourceManagerDataSource(
             GoogleCredential credential,
             Func<BaseClientService.Initializer, CloudResourceManagerService> factory,
-            string appName) : base(credential, factory, appName) { }
+            string appName) : base(credential, factory, appName)
+        { }
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension.DataSources/ResourceManagerDataSource.cs]: I'm not sure I like the look of this. I would prefer either empty brackets at the end of the line (like before) or each bracket on its own line.

File: GoogleCloudExtension/GoogleCloudExtension/TemplateWizards/Dialogs/ProjectIdDialog/PickProjectIdViewModel.cs [hunk 3]
```diff
@@ -108,7 +108,8 @@ public AsyncProperty LoadTask
         }
 
         public PickProjectIdViewModel(IPickProjectIdWindow owner)
-            : this(owner, DataSourceFactories.CreateResourceManagerDataSource, ManageAccountsWindow.PromptUser) { }
+            : this(owner, DataSourceFactories.CreateResourceManagerDataSource, ManageAccountsWindow.PromptUser)
+        { }
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/TemplateWizards/Dialogs/ProjectIdDialog/PickProjectIdViewModel.cs]: Same here. If we don't want empty brackets to end a line, then each bracket should have its own line.

File: GoogleCloudExtension/GoogleCloudExtensionUnitTests/CloudExplorerSources/PubSub/TopicViewModelBaseTests.cs [hunk 4]
```diff
@@ -186,13 +186,13 @@ private class TestTopicViewModelBase : TopicViewModelBase
             public TestTopicViewModelBase(
                 IPubsubSourceRootViewModel owner,
                 ITopicItem item,
-                IEnumerable<Subscription> subscriptions) : base(owner, item, subscriptions) { }
+                IEnumerable<Subscription> subscriptions) : base(owner, item, subscriptions)
+            { }
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtensionUnitTests/CloudExplorerSources/PubSub/TopicViewModelBaseTests.cs]: Here too.

File: GoogleCloudExtension/GoogleCloudExtension.TemplateWizards/GoogleProjectTemplateWizard.cs [hunk 5]
```diff
@@ -19,5 +20,5 @@ namespace GoogleCloudExtension.TemplateWizards
     /// of problems loading wizards from MEF component assemblies.
     /// </summary>
     public class GoogleProjectTemplateWizard : DelegatingTemplateWizard<IGoogleProjectTemplateWizard>
-    {}
+    { }
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension.TemplateWizards/GoogleProjectTemplateWizard.cs]: And here I see I did what I am complaining about myself.

File: GoogleCloudExtension/GoogleCloudExtension.DataSources/ResourceManagerDataSource.cs (hunk 2 above)
Comment (ivannaranjo) [reply GoogleCloudExtension/GoogleCloudExtension.DataSources/ResourceManagerDataSource.cs]: This is standard .NET coding guidelines, which is what we follow.

File: GoogleCloudExtension/GoogleCloudExtension/TemplateWizards/Dialogs/ProjectIdDialog/PickProjectIdViewModel.cs (hunk 3 above)
Comment (ivannaranjo) [reply GoogleCloudExtension/GoogleCloudExtension/TemplateWizards/Dialogs/ProjectIdDialog/PickProjectIdViewModel.cs]: This is the .NET coding guidelines that we follow.

**Pull request merged**